CMakeLists.txt

```diff
@@ -517,6 +520,14 @@ if(BUILD_TESTING)
     add_h3_test_with_arg(testH3NeighborRotations src/apps/testapps/testH3NeighborRotations.c 0)
     add_h3_test_with_arg(testH3NeighborRotations src/apps/testapps/testH3NeighborRotations.c 1)
     add_h3_test_with_arg(testH3NeighborRotations src/apps/testapps/testH3NeighborRotations.c 2)
+
+    # The "Exhaustive" part of the test name is used by the Valgrind CI test to exclude
+    # these files in .travis.yml.
```

Comment:
Nit: This comment is slightly too specific - I'd go with `The "Exhaustive" part of the test name is used to exclude the test from the test-fast target.`

Which makes me wonder whether this shouldn't just be `-Slow`, but admittedly `-Exhaustive` is more descriptive of what's currently there.

Reply:
I adjusted the comment to say that the "Exhaustive" part is for "test-fast" but I kept a comment that "test-fast" is there so that the Valgrind CI test doesn't timeout.